ruby-magic-link-example/views/index.erb

```diff
@@ -1,38 +1,52 @@
 <% if @current_user %>
 
-  <div class="logged_in_nav">
-  <div class="flex">
-      <p>You're logged in <%= @email %>, welcome!</p>
-  </div>
-  <div>
-      <img src="<%= url("/workos_logo_new.png")%>" alt="workos logo">
-  </div>
-</div>
-<div class='flex'>
-  <div class="logged_in_div_left">
-      <div>
-          <h1>Your app,</h1>
-          <h2>Enterprise Ready</h2>
-      </div>
-      <div>
-          <a href="https://workos.com/" target="_blank"><button class='button'>WorkOS</button></a>
-          <a href="https://workos.com/docs" target="_blank"><button class='button'>Documentation</button></a>
-          <a href="https://workos.com/docs/reference" target="_blank"><button class='button'>API Reference</button></a>
-          <a href="https://workos.com/blog" target="_blank"><button class='button'>Blog</button></a>
-          
-      </div>
-  </div>
+<div class="logged_in_nav">
+        <div class="flex">
+            <div>
+                <img src="<%= url("images/workos-logo-with-text.png")%>" alt="workos logo">
+            </div>
+            <div class="flex">
+                <p>You're logged in <%= @email %>, welcome!</p>
```

Comment:
Could you remove the border around the logo and also remove the text next to it? 
![CleanShot 2022-08-12 at 12 39 37](https://user-images.githubusercontent.com/47123183/184423013-912868fd-3ca2-4637-97b3-6eafea1afbf4.png)

Reply:
the border is related to the caching issue with local testing with Sinatra.  Will remove the login message!
<img width="1651" alt="CleanShot 2022-08-15 at 08 53 32@2x" src="https://user-images.githubusercontent.com/96083517/184669783-1b1aa027-acfd-4ed1-838f-d92ea93e491a.png">